packages/core/src/codewhisperer/service/transformByQ/transformApiHandler.ts

```diff
@@ -109,6 +109,15 @@ export async function uploadArtifactToS3(
 ) {
     throwIfCancelled()
     try {
+        const uploadFileByteSize = (await fs.promises.stat(fileName)).size
+        getLogger().info(
+            `Uploading zip at %s with checksum %s using uploadId: %s and size %s kB`,
+            fileName,
```

Comment:
Nit: is this fileName a full (relative/absolute) path? This variable could be better named if so